### PR TITLE
AEA-1037: Review changes. Tidying up.

### PR DESCRIPTION
**Pull request by mfjarvis**

### CODE REVIEW

File: coordinator/src/models/fhir/medication-request.ts [hunk 1]
```diff
@@ -11,17 +10,17 @@ export enum CourseOfTherapyTypeCode {
...
     "Short course (acute) therapy"
   ),
-  CONTINUOUS: createCodeableConcept(
+  CONTINUOUS: common.createCodeableConcept(
     "http://terminology.hl7.org/CodeSystem/medicationrequest-course-of-therapy",
     CourseOfTherapyTypeCode.CONTINUOUS,
     "Continuous long term therapy"
   ),
-  CONTINOUS_REPEAT_DISPENSING: createCodeableConcept(
+  CONTINOUS_REPEAT_DISPENSING: common.createCodeableConcept(
```
Comment (dan-jones-iw) [inline coordinator/src/models/fhir/medication-request.ts:23]: `CONTINUOUS_`

File: coordinator/src/models/fhir/medication-request.ts [hunk 2]
```diff
@@ -11,17 +10,17 @@ export enum CourseOfTherapyTypeCode {
 }
 
 export const CourseOfTherapyType = Object.freeze({
```
Comment (dan-jones-iw) [inline coordinator/src/models/fhir/medication-request.ts:12]: seems like `Object.freeze` doesnt stop me from changing values within the codeable concept. for this to be useful wouldnt we like it to stop any changes at all?

Comment (mfjarvis) [reply coordinator/src/models/fhir/medication-request.ts:12]: It doesn't but neither would
```
const SOME_CONSTANT = {
  key1: "value1",
  key2: "value2"
}
```
so it's as "immutable" as any object constant

Comment (mfjarvis) [reply coordinator/src/models/fhir/medication-request.ts]: Replaced with equivalent top level consts

File: coordinator/src/models/fhir/medication-request.ts (hunk 1 above)
Comment (mfjarvis) [reply coordinator/src/models/fhir/medication-request.ts]: Fixed

**Review by dan-jones-iw — APPROVED** [review]

**Pull request merged**